### DISCUSSION TITLE
Expanding Loss Functions Module

### DISCUSSION THREAD

**Discussion by LazaroHurtado** [LazaroHurtado/micrograd-rs#4] [Ideas]
Loss functions depicts the topology of the loss surface which is detrimental to whether or not our model converges to an effective local minimum. To give developers more creative freedom we should expand the loss functions micrograd-rs has to offer. A great place to start is by looking at the [functions Pytorch has to offer](https://pytorch.org/docs/stable/nn.html#loss-functions) and implementing those.

Anyone is able to claim which function they will be working on
